assets/stylesheets/gu-sass/colours.scss

```diff
@@ -76,7 +76,12 @@ $gu-colours: (
 
 	// External content palette
 	external-main-1:        #1c6326,
-	external-support-1:     #a9af2b
+	external-support-1:     #a9af2b,
+
+	// Support palette
+	light-blue: #d4ebf8,
```

Comment:
These variable names are quite ambiguous with the built-in colours names for example `lightblue`. (more colour names [here](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value))

Reply:
https://trello.com/c/fTdg3qbK/914-better-names-for-the-support-palette-colours